### ISSUE TITLE
rule jsx-boolean-value is ignored

### ISSUE THREAD

**Issue by SimpleCookie** [Microsoft/typescript-tslint-plugin#52]
Hi, I'm not sure if my issue is a bug or something that I am doing wrong. But my lint-rule is being ignored. I want to use the shorthand notation but the linter keeps throwing errors at me for it. I am coding react with typescript and got the settings as shown below.

Any idea how I can make it work?

tsconfig.json
```
{
  "compilerOptions": {
    "outDir": "build",
    "module": "esnext",
    "target": "es5",
    "lib": ["es6", "dom", "es2016", "es2017"],
    "sourceMap": true,
    "allowJs": false,
    "jsx": "react",
    "inlineSources": false,
    "declaration": true,
    "moduleResolution": "node",
    "forceConsistentCasingInFileNames": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "suppressImplicitAnyIndexErrors": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "plugins": [
      {
        "name": "typescript-tslint-plugin",
        "alwaysShowRuleFailuresAsWarnings": false,
        "ignoreDefinitionFiles": true,
        "configFile": "./tslint.json",
        "suppressWhileTypeErrorsPresent": false,
        "mockTypeScriptVersion": false
      }
    ]
  },
  "include": ["src"],
  "exclude": ["node_modules", "build", "dist", "example", "rollup.config.js"]
}
```

tslint.json

```
{
  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
  "linterOptions": {
      "exclude": [
        "config/**/*.js",
        "node_modules/**/*.ts"
      ]
    },
    "rules":  {
      "ordered-imports": [false, {
        "import-sources-order": "any",
        "named-imports-order": "case-insensitive",
        "grouped-imports": true
      }],
      "interface-name": [true, "never-prefix"],
      "jsx-no-lambda": false,
      "class-name": true,
      "member-access": false,
      "no-unused-expression": [true, "allow-fast-null-checks"],
      "no-var-requires": false,
      "trailing-comma": true,
      "triple-equals": true,
      "no-submodule-imports": false,
      "no-console": false,
      "tslint-no-circular-imports": false,
      "object-literal-shorthand": true,
      "forin": true,
      "arrow-return-shorthand": true,
      "jsx-boolean-value": "never",
      "object-literal-sort-keys": false
    }
  }
```

**Comment by mjbvz** [comment]
If you run tslint from the command line, does it show the same error?

**Comment by mjbvz** [comment]
Closing as this needs more information to investigate

**Comment by SimpleCookie** [comment]
> If you run tslint from the command line, does it show (or rather not show) the same error?

Sorry for late reply.
Running it from the command line results in no errors and no warnings. So oddly enough, it's only in vscode. I tried reloading the plugin and even reinstalling the plugin.

**Comment by SimpleCookie** [comment]
It's okay, after researching why it worked in my isolated environment and not in my project, I came to the conclusion, that I had to manually install `tslint-react`. I guess CRA normally does this for me, but this project didn't use CRA.
Thank you for the help and my apologies for the time wasted due to my/our mistake :P

The issue can remain closed. Although, it might be a good reference if someone else has the same problem in the future.
